app/javascript/admin/controllers/article_controller.js

```diff
@@ -1,7 +1,8 @@
 import { Controller } from 'stimulus';
 
 export default class ArticleController extends Controller {
-  static targets = ['featuredNumber'];
+  static classes = ['bgHighlighted', 'borderHighlighted'];
+  static targets = ['featuredNumber', 'cardBody'];
```

Comment:
added `cardBody` as it's definitely a targeted element (we highligh it), thus it's important to the controller